.gitignore

```diff
@@ -1,4 +1,4 @@
 coverage/
 node_modules/
 npm-debug.log
-package-lock.json
```

Comment:
Please add this back and remove the `package-lock.json` file. It was ignored on purpose :)

Reply:
Ok, added again